### DISCUSSION TITLE
MINIO_STORAGE_CLASS_STANDARD

### DISCUSSION THREAD

**Discussion by zhl19911203** [minio/minio#20581] [Q&A]
MINIO_STORAGE_CLASS_STANDARD，Can the value be set to EC:0?

Set this parameter to 0 to report an error Unable to initialize backend: failed to load rebalance data: Storage resources are insufficient for the read operation .minio.sys/rebalance.bin

**Comment by AlexZIX** [comment]
According to [this doc](https://min.io/docs/minio/linux/reference/minio-server/settings/storage-class.html) you can set EC:0.
The minimum supported value is 0, which indicates no erasure coding protections. These deployments rely entirely on the storage controller or resource for availability / resiliency.

It seems that your error not related to storage class.
